nc-news/src/Utils/api.js

```diff
@@ -4,8 +4,12 @@ const api = axios.create({
   baseURL: 'https://be-nc-news-ttz6.onrender.com/api',
 })
 
-export const fetchArticles = (sortBy = 'created_at', order = 'desc') => {
-  return api.get('/articles')
+export const fetchArticles = (topic, sortBy = 'created_at', order = 'desc') => {
+  let url = `/articles?sort_by=${sortBy}&order=${order}`
+  if (topic) {
+    url += `&topic=${topic}`
+  }
+  return api.get(url)
```

Comment:
You've taken the long way around here. Have a look at the optional params object that you can use in Axios. You can read more in the notes on Axios in NC notes. It will clean this up for you 😄